nginx.conf

```diff
@@ -73,6 +74,13 @@ http {
         location /static/ {
             alias static/;
         }
+
+        # nginx server status used for monitoring
+        location /nginx_status {
+            stub_status on;
```

Comment:
This is giving me an error that prevents the server to start in local:

nginx: [emerg] unknown directive "stub_status" in /home/renard/Code/Lua/snapCloud/nginx.conf.compiled:85

Reply:
To fix this we'd need to recompile nginx by hand, which is a no-go. We want to stay in sync with the official OpenResty build. Sorry but I'm taking this directive out...

Reply:
But it is in openresty. It's in the version that's on the server and it's in the openresty I installed locally from their docs.

Reply:
`root@snap-staging:~# openresty -V | grep stub_status
nginx version: openresty/1.13.6.1`

(Which has openresty simply from running prereqs.sh and no other config changes.)

Reply:
Oh you're right, I don't know why my local copy didn't have the proper OpenResty version.

I'll revert that.

Reply:
No worries. :)

Probably a good thing to make sure we're all on the same version anyway. ;)